app/controllers/LeaveFeedbackController.scala

```diff
@@ -60,14 +63,16 @@ trait LeaveFeedbackController extends AtedBaseController with AtedFrontendAuthHe
     ))
   }
 
-  def thanks(returnUri: String) = AuthAction(AtedSubscriptionNotNeededRegime) {
-    implicit atedContext =>
+  def thanks(returnUri: String) : Action[AnyContent] = Action.async { implicit request =>
```

Comment:
RED - This needs to be the less strict authAction.

Reply:
Replaced with authorisedForNoEnrolments